.github/workflows/main.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           go-version: ${{ env.GOVERSION }}
       # Download go-swagger
       - name: download go-swagger
-        run : go get -u github.com/go-swagger/go-swagger/cmd/swagger@master
+        run : cp go.* $HOME && go get -u github.com/go-swagger/go-swagger/cmd/swagger@master && cp $HOME/go.* .
```

Comment:
Is this needed just for the go.mod/go.sum stuff?

Reply:
yes

Reply:
I **think** there's a way to do this with something like `go -mod=readonly` but this is fine for now.